src/bp/nlu-core/predict-pipeline.ts

```diff
@@ -68,31 +68,37 @@ interface E1IntentPred {
 const DEFAULT_CTX = 'global'
 const NONE_INTENT = 'none'
 
+// Gives detected language if found one
+// If language is in bot languages give it to predict
+// Else give default one
 async function DetectLanguage(
   input: PredictInput,
   predictorsByLang: _.Dictionary<Predictors>,
   tools: Tools
 ): Promise<{ detectedLanguage: string; usedLanguage: string }> {
   const supportedLanguages = Object.keys(predictorsByLang)
+  // Need to train before predicting language
```

Comment:
rm

Reply:
Ok, behaviour will be to predict language even if model is not trained which is totally coherent